src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -500,8 +531,13 @@ public void onError(ServerError error)
    
    private void persistHtmlPath(String htmlPath)
    {
+      persistDocumentProperty("htmlPath", htmlPath);
```

Comment:
Just wanted to confirm that the html paths are constructed so that they survive from session to session (since the R session temp directory gets blasted at the end of a session). These files probably want to live within the module_context::scopedScratchPath (per-project "state" file storage). @jmcphers Does that sound right to you?